src/main/components/DataTree.jsx

```diff
@@ -15,12 +25,14 @@ const DataTree = (props) => {
   };
 
   return (
-    <section className='wrapper' id={`tree-${treeCount}`} style={{ maxHeight: '350px', overflow: 'auto' }} >
+    <section className='wrapper' id={`tree-${treeCount}`} >
```

Comment:
Where is the max height being set now ? I guess a better question is how is overflow being handled ?

Reply:
The `styles` object (lines 4–12) gets passed into the style prop on line 34—overflow is handled on lines 8–9